single-template.php

```diff
@@ -55,6 +55,13 @@
                                 <p><?php esc_html_e( 'Please provide an update by posting a comment.', 'disciple_tools' )?></p>
                             </div>
                         </a>
+                        <div class="bordered-box detail-notification-box show-for-large" style="background-color:#F43636">
+                                <h4>
+                                    <img src="<?php echo esc_html( get_template_directory_uri() . '/dt-assets/images/alert-circle-exc.svg' ) ?>"/>
+                                    <?php echo esc_html( sprintf( __( 'This %s needs an update.', 'disciple_tools' ), strtolower( $post_settings["label_singular"] ) ) ) ?>
+                                </h4>
+                                <p><?php esc_html_e( 'Please provide an update by posting a comment.', 'disciple_tools' )?></p>
+                            </div>
```

Comment:
Not the most elegant solution, but it works. If we can think of a way to do this without having to duplicate the notification box HTML that would be ideal, but we are doing the same things with menus and other places so it isn't the end of the world.

Reply:
Thanks! This works great